kitsune/search/v2/signals/questions.py

```diff
@@ -0,0 +1,64 @@
+from django.db.models.signals import post_save, post_delete, m2m_changed
+from django.dispatch import receiver
+from django.conf import settings
+from kitsune.search.v2.es7_utils import (
+    index_object,
+    delete_object,
+    index_objects_bulk,
+    remove_from_field,
+)
+from kitsune.questions.models import Question, QuestionVote, Answer, AnswerVote
+from taggit.models import Tag
+
+
+@receiver(post_save, sender=Question)
```

Comment:
Let's add to all the signals a UID 
https://docs.djangoproject.com/en/3.1/topics/signals/#preventing-duplicate-signals